teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(age, specialization, name = 'unknown')
+    super(age, name)
+    @specialization = specialization
+  end
```

Comment:
- [ ] in the initialize method in the `Teacher` class parent_permission should be included as well with a default value.